assets/js/load_more.js

```diff
@@ -3,10 +3,14 @@ jQuery(document).ready(function () {
 })
 
 function initLoadMore() {
+  var loadmore_active = false;
   jQuery('.jtf-load-more').on('click', function (e) {
     e.preventDefault();
+    if (loadmore_activiti) return false;
+    loadmore_active = true;
```

Comment:
i don't think you should set loadmore active each button click. it should be set on init, and then reset once last page reached.